docs/_tutorials/getting_started.md

```diff
@@ -187,25 +189,27 @@ app.message('hello', ({ message, say }) => {
 })();
 ```
 
-If you restart your app, you should be able to add your bot user to a channel, say "hello", and it will respond.
+If you restart your app, you should be able to add your bot user to a channel, send any message that contains "hello", and it will respond.
 
-This is a basic example, but it gives you a place to start customizing your app based on your end goal. Let's try something a little more interactive by sending an interactive button rather than plain text.
+This is a basic example, but it gives you a place to start customizing your app based on your end goals. Let's try something a little more interactive by sending a button rather than plain text.
 
 ---
 
 ### Sending and responding to actions
 
-To use features like buttons, select menus, datepickers, dialogs, or message actions, you’ll need to enable interactivity. Similar to how you set up your app to listen for events, you'll need to specify a URL for Slack to send the action (such as, someone clicked a button) to.
+To use features like buttons, select menus, datepickers, dialogs, and message actions, you’ll need to enable interactivity. Similar to events, you'll need to specify a URL for Slack to send the action (such as *user clicked a button*).
 
 Back on your app configuration page, click on **Interactive Components** on the left side. You'll see that there's another **Request URL** box.
 
-By default, Bolt is configured to use same endpoint for interactive components that it uses for events, so use the same request URL as above (in the example, it was `https://8e8ec2d7.ngrok.io/slack/events`). Press the **Save Changes** button in the lower right hand corner, and that's it. Your app is now all set up for interactivity!
+By default, Bolt is configured to use same endpoint for interactive components that it uses for events, so use the same request URL as above (in the example, it was `https://8e8ec2d7.ngrok.io/slack/events`). Press the **Save Changes** button in the lower right hand corner, and that's it. Your app is all set up for interactivity!
 
 ![Configuring a Request URL](../assets/request-url-config.png "Configuring a Request URL")
 
-Now, let's go back to your app's code and add our own interactivity. This will consist of two steps: first, your app will send a message that contains a button. Then, your app will set up a listener for when that button is clicked and do something in response.
+Now, let's go back to your app's code and add our own interactivity. This will consist of two steps:
+- First, your app will send a message that contains a button.
+- Next, your app will set up a listener that responds to a user when they click your app's button
```

Comment:
```suggestion
- Next, your app will listen to the action of a user clicking the button and respond
```

I want to intentionally use the word "action" so that it lines up with what the reader sees later, calling `app.action()`.